<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadBarcodes</class>
 <widget class="QDialog" name="UploadBarcodes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Загрузка штрих-кодов в систему</string>
  </property>
  <widget class="QLabel" name="L_ADDBARCODES">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить штрих-коды</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_FIRST">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_SECOND">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="L_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>От</string>
   </property>
  </widget>
  <widget class="QLabel" name="L_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>До</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_ADD">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="CB_TYPE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>74</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
